linkerd/app/core/src/proxy/buffer.rs

```diff
@@ -120,7 +120,7 @@ impl<M: Clone, D: Clone, Req> Clone for Make<M, D, Req> {
     }
 }
 
-impl<T, M, D, Req> svc::Service<T> for Make<M, D, Req>
+impl<T, M, D, Req> svc::Service<T> for MakeBuffer<M, D, Req>
```

Comment:
Thoughts on renaming `Self::Future` here `MakeBufferFuture` instead of keeping it `MakeFuture`?